src/server/utils/cluster-manager/cluster-manager.ts

```diff
@@ -63,6 +63,10 @@ function getSourceFromExternal(external: External): string {
   return String(external.source);
 }
 
+function externalContainsSource(external: External, source: string): boolean {
+  return Array.isArray(external.source) ? external.source.indexOf(source) > -1 : String(external.source) === source;
```

Comment:
We need to set ts target to version with `Array.includes`. That's task for another day!

Reply:
Yes, I was looking for an `Array` method for this, but could not find.